printer-linter/src/printerlinter/linters/formulas.py

```diff
@@ -0,0 +1,195 @@
+import difflib
+import json
+import os
+import re
+from configparser import ConfigParser
+from pathlib import Path
+from typing import Iterator
+
+from ..diagnostic import Diagnostic
+from ..replacement import Replacement
+from .linter import Linter
+
+FORMULA_NAMES = [
+    "extruderValue",
+    "extruderValues",
+    "anyExtruderWithMaterial",
+    "anyExtruderNrWithOrDefault",
+    "resolveOrValue",
+    "defaultExtruderPosition",
+    "valueFromContainer",
+    "extruderValueFromContainer",
+    "math",
```

Comment:
This is a module, not a function, right ?

Reply:
Hmm so it's the same for the other `math` methods. Then maybe rename all the functions to add their module ? like `math.log`, `math.atan`, `math.acos`, ...

Reply:
I have '.' as a delimiter for strings.